packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/ContextSelectorDemo/ContextSelectorDemo.tsx

```diff
@@ -53,7 +53,7 @@ export class ContextSelectorDemo extends React.Component<{}, ContextSelectorStat
       this.setState({ searchValue: value });
     };
 
-    this.onSearchButtonClick = event => {
+    this.onSearchButtonClick = () => {
```

Comment:
As a demo, it would be good to show what args can be are expected here

Reply:
Added argument back in